create/using-manual-approval.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
```

Comment:
Nicely done with the placement of the attributes include as this helps attributes render in the assembly title on the customer portal :+1: